src/types/ethereum.d.ts

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/// <reference types="react-scripts" />
+
+interface Window {
```

Comment:
Why is this needed? Please update the already existing global.d.ts file, with the new methods to added in App.tsx

Reply:
I added a TODO for metamask 2.0.0. This will solve the issue but let's not risk it before release